### DISCUSSION TITLE
Disable standby led (blue)

### DISCUSSION THREAD

**Discussion by drgkt** [arendst/Tasmota#12427] [Q&A] [answered]
It looks my smart plug (Athom T29) has 2 leds? A blue and a red.
A. Blue
1. Blinking when NOT connected to wifi
2. Steady ON when connected.

B. Red
1. Steady ON when device is on
2. OFF when device is off.

I want to change A2 to "OFF when connected" and keep others as are.

Your help is appreciated
![IMG_20210620_085011](https://user-images.githubusercontent.com/32744663/122748748-3a990900-d295-11eb-9951-43671605a6f4.jpg)
![IMG_20210620_085233](https://user-images.githubusercontent.com/32744663/122748756-3bca3600-d295-11eb-9f55-786b675a0bd6.jpg)
![IMG_20210620_085249](https://user-images.githubusercontent.com/32744663/122748757-3c62cc80-d295-11eb-9420-e65bc65f1e6e.jpg)
![IMG_20210620_085130](https://user-images.githubusercontent.com/32744663/122748760-3cfb6300-d295-11eb-87e1-ba404dd5c15e.jpg)

**Comment by barbudor** [comment]
Swap LedLink for LedLink_i in the template

**Reply by drgkt** [reply]
Can you please elaborate?
Below are present relative values (I played with, no success)
12:49:03.177 CMD: ledpower
12:49:03.183 RSL: RESULT = {"LedPower1":"OFF"}
12:49:21.168 CMD: ledstate
12:49:21.174 RSL: RESULT = {"LedState":1}
12:49:43.659 CMD: ledpwmmode
12:49:43.664 RSL: RESULT = {"LedPwmMode1":"OFF"}
12:50:04.835 CMD: setoption31
12:50:04.840 RSL: RESULT = {"SetOption31":"OFF"}

**Comment by barbudor** [comment, marked as answer]
On the template, where you configured the GPIOs
Menu configuration - Template

**Reply by drgkt** [reply]
Did not touch that since I got the device!
You mean the GPIO13 right?
![Template](https://user-images.githubusercontent.com/32744663/122754237-0ffe7e80-d29c-11eb-9094-adbe17b57a20.jpg)

**Reply by barbudor** [reply]
Yes, that's it. LedLink => LedLink_i will invert on and off states
So the led will still blink when loosing connection but will stay off when connected.
I do the same on all my devices (plugs, wall switches, ...)

**Reply by drgkt** [reply]
Done! 
Thank you SOOOOOO MUCH!

**Reply by barbudor** [reply]
no problem
While you puchased the plug pre-flashed and pre-cofigured, spend time reading the doc as you would have done when flashing it yourself. That will help you to understand Tasmota and get the best of it.
Enjoy

**Reply by KeilFelix** [reply]
Oh, it makes me so happy reading this. I had the same problem. My wife was annoyed of the blue light in the bedroom. Some other vendor locked in device would have been thrown away. It is so great to have full control over every aspect of the hardware without the need of exploits. Thank you @barbudor, Tasmota, and Athom!

**Reply by barbudor** [reply]
There is also a nice option : LedPWM
That allow you to define the brightness level for both ON and OFF state
I'm using that to keep a very very dim blue led ON when the room light if off which is not annoying but still allow to find the switch at night if needed.

**Comment by drgkt** [comment]
@barbudor
I sent you a message to your profile email.

**Reply by barbudor** [reply]
Why?
What would you ask that couldn't be asked publicly? 
I don't do private consulting 😉

**Comment by drgkt** [comment]
Well, it is off topic and I thought I would contact you there. I thought it was ok since you have it on your profile.
Forgive me. I will start another topic.
